README.md

```diff
@@ -40,3 +40,5 @@ To add or remove rules, first [submit an issue](https://github.com/digitronas/ak
 Clone this repository and run `composer install`. This will install ecs and grumphp. You might have to [bypass grumphp](https://github.com/phpro/grumphp/blob/master/doc/faq.md#how-can-i-bypass-grumphp), since it might not allow you to commit code if there are files that don't meet the standard.
 
 Create a pull request to `master` branch. The changes will be merged after they pass code review.
+
+If an issue should be closed once a pull request is merged, it can be done automatically using Github issue references. Add "Closes #{issue number}" to description ([example](https://github.com/digitronas/akeneo-coding-standard/pull/5)). References should not appear in pull request title or any of its commits.
```

Comment:
It would be good to point out to the specific version of the reference under the following:

> References should not appear in pull request title or any of its commits.

The `#number` is also a reference, more like these exists. Should we tolerate a reference of another type than `closes`? I see no harm for another type of references to be used in commits or description. Either way, none of the references should probably be used in the title, I may agree with this one. Feel free to decide on this topic if you have an argument to support your opinion.